src/Validator/Constraints/Password.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Validator\Constraints;
+
+use Symfony\Component\PasswordHasher\PasswordHasherInterface;
+use Symfony\Component\Validator\Constraints\Compound;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\NotCompromisedPassword;
+use Symfony\Component\Validator\Constraints\PasswordStrength;
+
+final class Password extends Compound
```

Comment:
smells like a class for core? Or our UserBundle?

Clever idea...

Reply:
It has a lot of potential customization that I wouldn't want to expose as bundle config for such a simple class